### DISCUSSION TITLE
Missing Equation due to KaTeX parse error

### DISCUSSION THREAD

**Discussion by benpymer** [Part-II-Computational-Physics/.github#31] [General]
Hi!

I think the initial definition of H_loc isn't visible in my exercise due to an error in the KaTeX code that's meant to display it. Here's a screenshot of the error message I'm getting. 

![Screenshot (31)](https://user-images.githubusercontent.com/125975405/221365316-2731efe6-c01f-429e-91c0-3c3990daac20.png)

I think I need it to complete task 1, so I'm struggling a bit! Would love to know what the equation is supposed to read! Thanks

**Comment by benpymer** [comment]
I found the issue by opening the editor for the markdown. The template has a full stop after the $$ at the end of the last equation. This is confusing the code and meaning it doesn't realise that $$ is the end of math mode, rather interpreting it as a function which you cannot call. I don't know how it got there or if anyone else is having the same issue, but it might be worth fixing it if they are? Having removed the full stop, my equation containing H_loc looks like this

![Screenshot (32)](https://user-images.githubusercontent.com/125975405/221366169-8c7f32fe-2653-42cf-80b4-a2d01bec7962.png)

**Comment by JNHallen** [comment]
Sorry the full stop did not cause any trouble for me, but it also does not need to be there. Thank you for pointing this out.
